packages/core/test/actor.test.ts

```diff
@@ -1151,10 +1254,12 @@ describe('actors', () => {
 
   it('should be able to spawn callback actors in (lazy) initial context', (done) => {
     const machine = createMachine<{ ref: ActorRef<any> }>({
-      context: () => ({
-        ref: spawnCallback((sendBack) => {
-          sendBack({ type: 'TEST' });
-        })
+      context: ({ spawn }) => ({
+        ref: spawn(
+          createDeferredBehavior(() => (sendBack) => {
```

Comment:
maybe we can rethink the name? i guess "callback" was pretty broad, but the same could be said about deferred and i guess people might have got used to "callback actors"

Reply:
Renamed to `createCallbackBehavior`